### PR TITLE
[MIG] jasper_reports: Migrated Jasper Report module for Odoo 15

### PR DESCRIPTION
**Pull request by ssaid**

### CODE REVIEW

**Comment by ssaid** [comment]
Hi thanks for this awesome module, i want to contribute with the migration for odoo 15.0.
This is working in v15, as the upstream does not have the branch 15.0 I've created the PR against 14.0, is this ok ?

**Comment by hiren-serpentcs** [comment]
Hi @ssaid 
Thank you for your contribution. Created a new branch [15.0](https://github.com/JayVora-SerpentCS/Jasperreports_odoo/tree/15.0), please change the PR target branch.

File: jasper_reports/JasperReports/jasper_server.py [hunk 1]
```diff
@@ -68,11 +68,13 @@ def start(self):
                               'view'))
         else:
             libraries = str(java_path) + '/lib'
-            if os.path.exists(str(libraries)):
-                self.javapath = java_path
-            else:
-                raise UserError(_('libraries Not Found !\n'
-                                  'No libraries found in Java'))
+            print(libraries)
```
Comment (hiren-serpentcs) [inline jasper_reports/JasperReports/jasper_server.py]: Please remove the print statements

File: jasper_reports/JasperReports/jasper_server.py [hunk 2]
```diff
@@ -68,11 +68,13 @@ def start(self):
...
+            self.javapath = java_path
+           # if os.path.exists(str(libraries)):
+           #     self.javapath = java_path
+           # else:
+           #     raise UserError(_('libraries Not Found !\n'
+           #                       'No libraries found in Java'))
```
Comment (hiren-serpentcs) [inline jasper_reports/JasperReports/jasper_server.py]: any reason to remove this?

Comment (ssaid) [reply jasper_reports/JasperReports/jasper_server.py]: None, it was for testing purposes

**Pull request merged**